public/locales/en/translation.json

```
@@ -124,6 +124,8 @@
     "odisha_youth_PPsCompletionMessage": "<div class='text-class'>Thank you for sharing — your journey matters. Keep going, you are creating real change.</div>",
     "shiksha_samvad_PPsCompletionMessage": "<div class='text-class'>Thank you so much for spending time and sharing your views.</div>",     
     "fgd_teacher_PPsCompletionMessage": "<div class='text-class'>Thank you for your time. Once we complete the analysis of the baseline report, we will share the relevant aspects with you</div>",
+    "stakeholder_fgd_PPsCompletionMessage": "<div class='text-class'>Thank you! Your response has been successfully submitted.</div>",
```

Biome (2.4.14)

[error] The key stakeholder_fgd_PPsCompletionMessage was already declared.

(lint/suspicious/noDuplicateObjectKeys)